test/system/120-load.bats

```diff
@@ -26,6 +26,13 @@ verify_iid_and_name() {
     is "$new_img_name" "$1"   "Name & tag of restored image"
 }
 
+@test "podman load invalid file" {
+    # Regression test for #9672 to make sure invalid input yields errors.
+    invalid=$PODMAN_TMPDIR/invalid
+    echo "I am an invalid file and should cause a podman-load error" > $invalid
+    run_podman 125 load -i $invalid
```

Comment:
Wish I'd seen this earlier. In future, I encourage you to check the error message. I'll open a PR to add that.

Reply:
>Shouldn't you be doing yoga on a mountain peak? :)

What makes you think I'm not? :wink: 

Ugly or not, the reason for error-message checking is to save yourself from typos or syntax errors:
```bash
    run_podman load -i $imvalid      <--- misspelled variable name, yields "flag needs an argument"
    run_podman load -i $nonexistent_path  <--- yields "no such file or directory"
```
...or other mistakes that throw an error for other reasons. (None of those are likely in this case, the code looks good, but an error-message check will greatly reassure future maintainers)